src/containers/post/post.component.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { voteForOptA, voteForOptB } from "database/votePost";
+
+const Post = (props) => {
+  console.log(props);
+  return (
+    <li className="rounded border-4 border-orange-600 mt-2">
+      <h6>{props.body}</h6>
+      {/* <p>{props.created_at}</p> */}
```

Comment:
Clean up comment

Reply:
leaving this in as a placeholder till i figure out the timestamping w firebase